src/sentry/ingest/transaction_clusterer/tasks.py

```diff
@@ -59,6 +62,7 @@ def spawn_clusterers(**kwargs: Any) -> None:
     time_limit=PROJECTS_PER_TASK * CLUSTERING_TIMEOUT_PER_PROJECT + 2,  # extra 2s to emit metrics
 )
 def cluster_projects(projects: Sequence[Project]) -> None:
+    pending = set(projects)
     num_clustered = 0
```

Comment:
We don't need to have both, but we don't know what's happening here so I'd rather have these two and see if there are any inconsistencies.